### DISCUSSION TITLE
Resume- Nikita Babbar

### DISCUSSION THREAD

**Discussion by NIKITA320495** [recodehive/resume-pitch#101] [Q&A]
[Resume-nikita babbar (1).pdf](https://github.com/user-attachments/files/17083875/Resume-nikita.babbar.1.pdf)

**Comment by sanjay-kv** [comment]
[ATS Version of CV (18).docx](https://github.com/user-attachments/files/17083897/ATS.Version.of.CV.18.docx)
[nikita.pdf](https://github.com/user-attachments/files/17083898/nikita.pdf)

**Reply by sanjay-kv** [reply]
@NIKITA320495

**Comment by NIKITA320495** [comment]
![Screenshot 2024-09-21 173012](https://github.com/user-attachments/assets/c3d45c4b-e4fd-4df6-90d6-b6b21764c513)
